test/Core.Test/AdminConsole/AutoFixture/OrganizationFixtures.cs

```diff
@@ -24,12 +24,14 @@ public void Customize(IFixture fixture)
     {
         var organizationId = Guid.NewGuid();
         var maxCollections = (short)new Random().Next(10, short.MaxValue);
+        var seats = (short)new Random().Next(100, short.MaxValue);
```

Comment:
100 is a bit arbitrary, the minimum seat value is 1 so I suggest using that.

Reply:
@eliykat I have added a new property to configure the `PlanType` on `OrganizationCustomization`. It defaults to `PlanType.EnterpriseAnnually` as that plan works for all the tests that use that customization.